docs/modeltester.rst

```diff
@@ -0,0 +1,45 @@
+Model Tester
+============
+
+.. versionadded:: 1.6
+
+``pytest-qt`` includes a fixture that helps testing
+`QAbstractItemModel`_ implementations. The implementation is copied
+from the C++ code as described on the `Qt Wiki <http://wiki.qt.io/Model_Test>`_,
+and it continuously checks a model as it changes, helping to verify the state
+and catching many common errors the moment they show up.
+
+Some of the conditions caught include:
+
+* Verifying X number of rows have been inserted in the correct place after the signal rowsAboutToBeInserted() says X rows will be inserted.
```

Comment:
:+1: I just copied it over from Qt Wiki and thought about adding `` while doing so, but ended up forgetting to actually do it, thanks!